Assets/MixedRealityToolkit-SDK/Features/Input/Scripts/SpeechInputHandler.cs

```diff
@@ -24,37 +25,19 @@ public class SpeechInputHandler : MonoBehaviour, IMixedRealitySpeechHandler
         [Tooltip("The keywords to be recognized and optional keyboard shortcuts.")]
         private KeywordAndResponse[] keywords = new KeywordAndResponse[0];
```

Comment:
I know this is not new to this PR, but do we really NOT want folks to be able to add keywords via script? Also, why an array instead of a List<>?